src/components/Roadmap.js

```diff
@@ -96,10 +102,16 @@ const Roadmap = () => {
           const implemented = issues
             .filter(
               (issue) =>
-                issue.state === "closed" &&
-                "allcontributors[bot]" !== issue.user.login &&
-                !!issue.pull_request
+              {issue.state === "closed" &&
+                "allcontributors[bot]" !== issue.user.login
+                 }
+              if(!!issue.pull_request.url){
+                return(isMerged(issue.pull_request.url))
+              }
+                
+                
```

Comment:
Syntax here is broken... 
```
.filter(
  (issue) => {
    /* code here is run without an automatic return */
  }
  /* below this is invalid syntax since we're outside the curly braces and still within the .filter() method */
  if (!!issue.pull_request.url) {
    return isMerged(issue.pull_request.url)
  }
)
```